### PR TITLE
railings less bad

### PR DESCRIPTION
**Pull request by Bibby0110**
no description, railings no blocky rods + throwy stuff over them

### CODE REVIEW

**Comment by Simplehorror** [comment]
the parent code does this shit already, but it doesn't work because the immovable rod isn't unstoppable for some reason. make the immovable rod unstoppable

**Comment by Simplehorror** [comment]
yeah it doesnt even compile. compared to my 1 line fix

**Comment by Bibby0110** [comment]
> yeah it doesnt even compile. compared to my 1 line fix

you don’t have a fix

File: code/game/objects/structures/railings.dm [hunk 1]
```diff
@@ -67,19 +67,20 @@
...
+	if(get_dir(loc, target) & dir)
+		var/checking = FLYING | FLOATING
+		return . || mover.throwing || mover.movement_type & checking
+	return TRUE
 
 /obj/structure/railing/corner/CanAllowThrough(atom/movable/mover, turf/target)
 	. = ..()
 	return TRUE
 
 /obj/structure/railing/CheckExit(atom/movable/O, turf/target)
```
Comment (Bibby0110) [inline code/game/objects/structures/railings.dm]:
```suggestion
/obj/structure/railing/CheckExit(atom/movable/mover, turf/target)
```

**Comment by Simplehorror** [comment]
> > yeah it doesnt even compile. compared to my 1 line fix
> 
> you don’t have a fix

the fix was working fine and yours is a webedit

**Comment by Bibby0110** [comment]
> > > yeah it doesnt even compile. compared to my 1 line fix
> > 
> > 
> > you don’t have a fix
> 
> the fix was working fine and yours is a webedit

so? mine is better, webedit or not

**Comment by Simplehorror** [comment]
> > > > yeah it doesnt even compile. compared to my 1 line fix
> > > 
> > > 
> > > you don’t have a fix
> > 
> > 
> > the fix was working fine and yours is a webedit
> 
> so? mine is better, webedit or not

no it isn't, it doesn't work

**Comment by Bibby0110** [comment]
> > > > > yeah it doesnt even compile. compared to my 1 line fix
> > > > 
> > > > 
> > > > you don’t have a fix
> > > 
> > > 
> > > the fix was working fine and yours is a webedit
> > 
> > 
> > so? mine is better, webedit or not
> 
> no it isn't, it doesn't work

sure it does

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**